include/ray/logging.h

```diff
@@ -14,7 +14,11 @@
   } else if (LEVEL == RAY_DEBUG) { \
     \
   } else { \
-    std::cout << MESSAGE << std::endl; \
+    if(global_ray_config.log_to_file) { \
+      global_ray_config.logfile << MESSAGE << std::endl; \
+    } else { \
+      std::cout << MESSAGE << std::endl; \
+    } \
```

Comment:
This is just for logging... IMO if it's fine to refer to `cout` then it's fine to refer to `global_ray_config` since both are global (and I think they're fine). Shouldn't the `cout`s be `cerr` though?